cmake/gcc-m-cpu.cmake

```diff
@@ -24,6 +24,12 @@ if("${ARCH}" STREQUAL "arm")
     else()
       set(GCC_M_CPU cortex-m33+nodsp)
     endif()
+  elseif(CONFIG_CPU_CORTEX_M55)
```

Comment:
Should this also handle a `+nofpu` option if the design is similar to the dual core AN524, where only core1 has the FPU and DSP options, or is `CONFIG_FPU` not being set sufficient?

Reply:
I feel like maybe we cross that bridge when we get there?  ie the AN547 is a single core so it supports all the features.  When we start seeing other actual SoCs with M55 and/or another FPGA image we can update this as needed